Sources/CryptorRSA/CryptorRSA.swift

```diff
@@ -615,7 +615,7 @@ public class CryptorRSA {
 
                 // Unlike other return values above, this return indicates if signature verifies or not
                 rc = signature.data.withUnsafeBytes({ (sig: UnsafePointer<UInt8>) -> Int32 in
-                    return EVP_DigestVerifyFinal(md_ctx, sig, signature.data.count)
+                    return SSL_EVP_digestVerifyFinal_wrapper(md_ctx, sig, signature.data.count)
```

Comment:
Could we add a comment above this line explaining where this wrapper is defined (and briefly why we need it)? Something like:
```
// Wrapper for OpenSSL EVP_DigestVerifyFinal function defined in
// IBM-Swift/OpenSSL/shim.h, to provide compatibility with OpenSSL
// 1.0.1 and 1.0.2 on Ubuntu 14.04 and 16.04, respectively.